sql/moz-fx-data-shared-prod/accounts_backend_derived/users_first_seen_v1/query.sql

```diff
@@ -0,0 +1,41 @@
+WITH users_services_daily_new_entries AS (
+  SELECT DISTINCT
+    user_id_sha256,
+    country,
+    seen_in_tier1_country,
+    registered
+  FROM
+    accounts_backend.users_services_daily
```

Comment:
Wouldn't it be more efficient to use users_services_first_seen instead for this table and reprocessing all the clients inside `users_services_daily`?